x-pack/plugins/security_solution/public/management/cypress/e2e/automated_response_actions/form.cy.ts

```diff
@@ -147,62 +192,80 @@ describe(
         cy.intercept('PUT', '/api/detection_engine/rules').as('deleteResponseAction');
         cy.getByTestSubj('ruleEditSubmitButton').click();
         cy.wait('@deleteResponseAction').should(({ request }) => {
-          expect(request.body.response_actions).to.be.equal(undefined);
+          expect(request.body.response_actions.length).to.be.equal(2);
         });
         cy.contains(`${ruleName} was saved`).should('exist');
       });
     });
 
-    describe('User should not see endpoint action when no rbac', () => {
-      const [ruleName, ruleDescription] = generateRandomStringName(2);
+    describe(
+      'User should not see endpoint action when no rbac',
+      {
+        tags: [
+          // Not supported in serverless! Test suite uses custom roles
+          '@brokenInServerless',
+        ],
+      },
+      () => {
+        const [ruleName, ruleDescription] = generateRandomStringName(2);
 
-      beforeEach(() => {
-        login(ROLE.endpoint_response_actions_no_access);
-      });
+        beforeEach(() => {
+          login(ROLE.endpoint_response_actions_no_access);
```

Comment:
can you use a pre-defined role (one that is supported in serverless) that does not have access to response actions? Maybe a T1 or T2 analyst?

Reply:
I think I tried once, but T1/T2 was missing some rules permissions. I'll try again 👍

Reply:
Role.Rule_author works as we need. So I will use this one :)